src/c++/perf_analyzer/genai-pa/llm_inputs/llm_inputs.py

```diff
@@ -0,0 +1,282 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from copy import deepcopy
+from typing import Dict, List, Tuple
+
+import requests
+from genaipa_exceptions import GenAiPAExceptions
+from requests import Response
+
+
+class LlmInputs:
+    """
+    A library of methods that control the generation of LLM Inputs
+    """
+
+    OUTPUT_FILENAME = "./llm_inputs.json"
+
+    OPEN_ORCA_URL = "https://datasets-server.huggingface.co/rows?dataset=Open-Orca%2FOpenOrca&config=default&split=train"
+    CNN_DAILYMAIL_URL = "https://datasets-server.huggingface.co/rows?dataset=cnn_dailymail&config=1.0.0&split=train"
+
+    DEFAULT_STARTING_INDEX = 0
+    MINIMUM_STARTING_INDEX = 0
+
+    DEFAULT_LENGTH = 100
+    MINIMUM_LENGTH = 1
+
+    EMPTY_JSON_IN_PA_FORMAT = {"data": [{"payload": []}]}
```

Comment:
FYI the payload key is only true for OpenAI

Reply:
Changed name to reflect this